spec/javascripts/components/cookie-banner-spec.js

```diff
@@ -161,15 +161,15 @@ describe('Cookie banner', function () {
     new GOVUK.Modules.CookieBanner(element).init()
 
     var acceptCookiesButton = document.querySelector('[data-accept-cookies]')
-    var mainCookieBanner = document.querySelector('.gem-c-cookie-banner__message')
+    var mainCookieBanner = document.querySelector('.js-banner-wrapper')
```

Comment:
Just curious why this class changed?

Reply:
@andysellick I can change it back - basically it was checking the element was hidden but the actual thing that gets hidden is it's parent element. Before I used `!this.isElementVisible` to check if something is hidden the test was failing but I reckon it will still work if I change it back to `.gem-c-cookie-banner__message`